docs/antora/modules/ROOT/pages/Common_Project_Layouts.adoc

```diff
@@ -103,6 +109,11 @@ latter of which runs your code on Node.js, which must be pre-installed)
 `ScalaJSModule` also exposes the `foo.fastOpt` and `foo.fullOpt` tasks for
 generating the optimized Javascript file.
 
+ScalaJSModule requires declaring it's dependencies as third party
+libraries (note the two sets of double-colons `::` necessary between the library name and the version).
+
+If you are following the [Scala.js Tutorial](http://www.scala-js.org/doc/tutorial/basic/), in the section where you test the code, it's mentioned that you need to make the DOM available to Node.js. Install the `jsdom` node library as instructed then add `def jsEnvConfig = T(JsEnvConfig.JsDom())` to your `build.sc` object. In Scala 3, you don't need the "scalajs-env-jsdom-nodejs" mentioned in the tutorial.
```

Comment:
This should be an Asciidoc link and https.

```suggestion
If you are following the https://www.scala-js.org/doc/tutorial/basic/[Scala.js Tutorial], in the section where you test the code, it's mentioned that you need to make the DOM available to Node.js. Install the `jsdom` node library as instructed then add `def jsEnvConfig = T(JsEnvConfig.JsDom())` to your `build.sc` object. In Scala 3, you don't need the "scalajs-env-jsdom-nodejs" mentioned in the tutorial.
```

Reply:
Can you add, why this is not needed in Scala 3? Is it already included or no longer necessary?